### DISCUSSION TITLE
Is Codespaces available for cellphones?<title>

### DISCUSSION THREAD

**Discussion by biban99** [community/community#45548] [Codespaces]
### Select Topic Area

Product Feedback

### Body

HI, does anyone knows if the codespaces can be run in the cell? Im not finding the way in iphone or ipad.

**Comment by itzsebas121** [comment]
Sí, puedes usar GitHub Codespaces en un iPhone o iPad, pero no directamente como en una laptop o PC. No existe una app nativa ni soporte completo para navegadores móviles, así que no podrás abrir el entorno de desarrollo completo desde Safari o Chrome en iOS.
Usar GitHub Codespaces con una app de terminal remota, como Blink Shell
, conectándote vía SSH al contenedor del Codespace.

O bien, usar un navegador con modo “escritorio completo”, aunque la experiencia suele ser limitada (teclado, rendimiento, etc.).

**Reply by thirmeth216-sudo** [reply]
45548
